src/stores/business.ts

```diff
@@ -437,6 +456,46 @@ export const useBcrosBusiness = defineStore('bcros/business', () => {
     return false
   }
 
+  /**
+ * Creates a new comment
+ * @param url the full URL to fetch the documents
+ * @returns the fetch documents object or throws error
+ */
+  const postComment = async (businessId: string, comment: CreateCommentI) => {
+    const apiURL = useRuntimeConfig().public.legalApiURL
+    const url = `${apiURL}/businesses/${businessId}/comments`
+    return await useBcrosFetch<{ comment: CommentIF }>(url, { method: 'POST', body: { comment } })
+      .then(({ data, error }) => {
+        if (error.value || !data.value) {
+          console.warn('postComment() error - invalid response =', error?.value)
+          throw new Error(error.value.message)
+        }
+        return data?.value
+      })
+  }
+  const createCommentBusiness = async (businessId: string, comment: string): Promise<CommentIF> => {
```

Comment:
I think this should just use the currentBusinessIdentifier? I don't think we want the possibility for accidentally posting comments to other businesses

Reply:
Done